javascript/features/killtime/killtime_commands.js

```diff
@@ -13,14 +13,15 @@ class KilltimeCommands {
         server.commandManager.buildCommand('killtime')
             .restrict(Player.LEVEL_ADMINISTRATOR)
             .sub('start')
-                .parameters([{ name: 'minutes', type: CommandBuilder.NUMBER_PARAMETER, optional: true }])
+                .parameters([{ name: 'minutes', type: CommandBuilder.NUMBER_PARAMETER, optional: true },
+                            { name: 'weapon', type: CommandBuilder.NUMBER_PARAMETER, optional: true}])
                 .build(KilltimeCommands.prototype.onKilltimeStartCommand.bind(this))
             .sub('stop')
                 .build(KilltimeCommands.prototype.onKilltimeStopCommand.bind(this))
             .build(KilltimeCommands.prototype.onKilltimeCommand.bind(this));
     }
 
-    onKilltimeStartCommand(player, minutes = 2) {
+    onKilltimeStartCommand(player, minutes = 2, weapon) {
```

Comment:
@RussellLVP Shouldn't the weapon-parameter also be defaulted here? I am not sure how the commandManager handles this to be honest.

Reply:
This results into `undefined` by dfault.